src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big-js';
+
+const operate = (first, second, operation) => {
+  const a = Big(first);
+  const b = Big(second);
+
+  switch (operation) {
+    case '+':
+      return a.plus(b).toString();
+    case '-':
+      return a.plus(b).toString();
+    case '×':
+      return a.times(b).toString();
+    case '÷':
+      if (b === '0') return '0';
+      return a.div(b).toString();
+    default:
```

Comment:
Guess you're missing `case %` and `case +/-` right?
Kindly add them.

Reply:
Those are taken care of by the calculate module since they do not require two operators